### ISSUE TITLE
Not sensitive enough?

### ISSUE THREAD

**Issue by jedwards1211** [hosembafer/react-swipe-to-dismiss#4]
I was just playing with the demo.  This was admittedly on my laptop with a touchpad, but it was sort of hard to swipe hard enough to dismiss the rows.  From what I can tell Hammer.js default swipe detection is more sensitive.

**Comment by hosembafer** [comment]
@jedwards1211 , you are right. I developed this only for desktop but will be nice to have it for mobile and tablet devices. Maybe investigation is needed; is there any lightweight alternatives of Hammer.js? PR welcomed.

**Comment by creationator** [comment]
I was trying to use the swipe-to-dismiss npm package for my react app. I noticed that you were one of the collaborators on it. I was reaching out because I was experiencing an error when I tried using it. For the most part, the functionality worked as expected but only one problem, once I swiped my component off the screen the element wasn't actually deleted from the dom (It just left an empty space) and returned an error of "TypeError: Cannot read property 'props' of undefined" See gif below.

I've tried a couple of other solutions such as placement, removing 'this' and even 'props'. I went ahead as to so far add the absolute path to the node_modules  directory but nothing works. I was wondering if anyone else using this had experienced this issue and if so what was there fix.
